google/cloud/storage/internal/grpc_notification_metadata_parser.h

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_STORAGE_INTERNAL_GRPC_NOTIFICATION_METADATA_PARSER_H
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_STORAGE_INTERNAL_GRPC_NOTIFICATION_METADATA_PARSER_H
+
+#include "google/cloud/storage/notification_metadata.h"
+#include "google/cloud/storage/version.h"
+#include <google/storage/v2/storage.pb.h>
+
+namespace google {
+namespace cloud {
+namespace storage {
+GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
+namespace internal {
+
+struct GrpcNotificationMetadataParser {
+  static NotificationMetadata FromProto(
+      google::storage::v2::Notification const& rhs);
+  static google::storage::v2::Notification ToProto(
+      NotificationMetadata const& rhs, std::string const& bucket_name);
+};
```

Comment:
Nit:  We say "This project follows the [Google Style Guide]", and it says, "Do not use a class simply to group static members."  Reconcile as you see fit.

Reply:
Removed the structs though that leaves a lot of other code with them.